src/config.ts

```diff
@@ -119,6 +119,7 @@ export class Config {
   public default = DEFAULT_SETTING
 
   public apihost = process.env.WECHATY_APIHOST || DEFAULT_SETTING.DEFAULT_APIHOST
+  public customizeServerHost = process.env.WECHATY_CUSTOM_SERVER_HOST || ''
```

Comment:
How about let's consider another environment variable name: `WECHATY_PUPPET_SERVICE_HOST`?

It will be a perfect match to the `WECHATY_PUPPET_SERVCIE_PORT` env name.

See:

https://github.com/wechaty/wechaty/blob/040784c94a137ae9da7fbea8095a9b5d2b385e5c/bin/io-client.ts#L77-L78